docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -170,218 +179,288 @@ Example
+- ``label_df``: Pandas DataFrame containing the target labels used in fitting the ridge models
+- ``sample_blocks``: Dictionary containing a mapping of sample block IDs to a list of corresponding sample IDs
 - ``covariates``: Pandas DataFrame containing covariates to be included in every model in the stacking
-  ensemble (optional).
+  ensemble (optional)
+
+.. _model_df:
 
 Return
 ------
 
 The fields in the model DataFrame are:
 
-- ``header_block``: An ID assigned to the header block *x0* corresponding to the coefficients in this row.
-- ``sample_block``: An ID assigned to the sample block *x0* corresponding to the coefficients in this row.
-- ``header``: The name of a column from the conceptual matrix *X0* that correspond with a particular row from the
-  coefficient matrix *B*.
-- ``alphas``: List of alpha names corresponding to the columns of *B*.
-- ``labels``: List of label (i.e., phenotypes) corresponding to the columns of *B*.
-- ``coefficients``: List of the actual values from a row in *B*.
+- ``header_block``: An ID assigned to the block :math:`x_0` to the coefficients in this row
+- ``sample_block``: An ID assigned to the block :math:`x_0` containing the group of samples represented on this row
+- ``header``: The column name in the conceptual genotype matrix :math:`X_0` that corresponds to a particular row in the coefficient matrix :math:`B`
+- ``alphas``: List of alpha names corresponding to the columns of :math:`B`
+- ``labels``: List of labels (i.e., phenotypes) corresponding to the columns of :math:`B`
+- ``coefficients``: List of the actual values from a row in :math:`B`
+
+Example
+-------
+
+.. code-block:: python
+
+    model_df = reducer.fit(block_df, label_df, sample_blocks, covariates)
 
-Model transformation
-====================
+3. Model transformation
+=======================
 
-After fitting, the ``RidgeReducer.transform`` method can be used to generate *X1* from *X0*.
+After fitting, the ``RidgeReducer.transform`` method can be used to generate :math:`X_1` from :math:`X_0`.
 
 Parameters
 ----------
 
-- ``block_df``: Spark DataFrame representing the beginning block matrix.
-- ``label_df``: Pandas DataFrame containing the target labels used in fitting the ridge models.
-- ``sample_blocks``: Dictionary containing a mapping of sample block IDs to a list of corresponding sample IDs.
-- ``model_df``: Spark DataFrame produced by the RidgeReducer fit method, representing the reducer model.
+- ``block_df``: Spark DataFrame representing the beginning block matrix
+- ``label_df``: Pandas DataFrame containing the target labels used in fitting the ridge models
+- ``sample_blocks``: Dictionary containing a mapping of sample block IDs to a list of corresponding sample IDs
+- ``model_df``: Spark DataFrame produced by the ``RidgeReducer.fit`` function, representing the reducer model
 - ``covariates``: Pandas DataFrame containing covariates to be included in every model in the stacking
```

Comment:
Not really new to this update, but should we not call this value `covariate_df` to be consistent with the other parameters? In the diagram as well.

Reply:
Changed to ``covariate_df`` in text, diagram, and notebooks